FrontEnd/src/components/SearchPage/Search.jsx

```diff
@@ -97,21 +95,21 @@ export function Search({ isAuthorized, userData }) {
                 searchPerformed={searchPerformed}
                 displayedResults={displayedResults}
                 isAuthorized={isAuthorized}
-                userData={userData}
               />
               <br />
             </>
-          ) : ( <div>
-          <br />
-            <p className={styles['search_result_error']}>
-              Пошук не дав результатів: компанії з іменем{' '}
-              <span className={styles['.search_result_error']}>
-                {searchTerm}
-              </span>{' '}
-              не було виявлено на даний момент
-            </p>
-             <br />
-          </div>
+          ) : (
+            <div>
+              <br />
```

Comment:
Do we need `br` here? It seems that we use it for margin here. 
Check

Reply:
Thank you for feedback. Done